src/io/executables/iges_to_irit.cc

```diff
@@ -36,7 +36,6 @@ int main(int argc, char *argv[]) {
     io::IGESReader iges_reader;
     splines = iges_reader.ReadFile(log.GetInput());
   } catch (std::runtime_error &error) {
-    std::cerr << error.what() << std::endl;
```

Comment:
I just wanted to make this file equal to the other io executable files. The `catch` case should be there because there can be more detailed error descriptions.

Reply:
So right now you are only passing on the error but without handling it, right? Is this the way it should be done? I would expect to handle the error as soon as it is caught, because it will get more and more difficult to understand it or provide information on it the further one goes up in the call graph.